Nodejs/Setup/NodejsTools/NodejsTools.wxs

```diff
@@ -18,11 +18,17 @@
                 <Directory Id="Dir_$(var.key)TestFrameworksMocha" Name="Mocha" />
             </Directory>
             <Directory Id="Dir_$(var.key)InstallLocationScripts" Name="Scripts">
-                <Directory Id="Dir_$(var.key)InstallLocationScriptsTypings" Name="Typings">
+                <Directory Id="Dir_$(var.key)InstallLocationScriptsTypings" Name="Typings">                
                     <Directory Id="Dir_$(var.key)ScriptsTypingsNode" Name="Node" />
                 </Directory>
             </Directory>
             <Directory Id="Dir_$(var.key)RemoteDebug" Name="RemoteDebug"/>
+            <Directory Id="Dir_$(var.key)Snippets" Name="Snippets">
+                <Directory Id="Dir_$(var.key)Snippets_1033" Name="1033">
+                <Directory Id="Dir_$(var.key)Snippets_1033_Nodejs" Name="Nodejs"/>
+                <!--<Directory Id="Dir_$(var.key)Snippets_1033_Test" Name="Test"/>-->
```

Comment:
For test snippets, it would be useful to take a look at Mocha and ExportRunner which are the two test frameworks we support. 

And speaking of tests, we should add some unit tests to verify all the core parts of this experience are working and also make sure we don't break it going forward. :smiley: